andela/ah-django
# #164857206 Travis Integration and Badge

#### What does this PR do?
Update ReadMe 
Add Travis-ci badge 

#### Description of the Tasks to be accomplished
-Add Travis ci integration 
-Add CI badge to Readme 
#### How should this be manually tested?
Go to https://github.com/rfpremier/ah-django/tree/ch/164857206-travis-ci-badge to see the badge

#### What are the relevant pivotal tracker stories?
[#164857206](https://www.pivotaltracker.com/story/show/164857206)

#### Screenshots
N/A
